### PR TITLE
data:Laxmankohar

### PR DESCRIPTION
**Pull request by Laxmankohar**
## Changes proposed
I have added my own data inside data folder.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3269"></a>

### CODE REVIEW

File: data/Laxmankohar.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/laxman-kohar-a85a9a195",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/laxmankohar92/",
+      "icon": "FaInstagram"
+    }
+
```
Comment (razlq) [inline data/Laxmankohar.json:28]: **Can you delete line number 28?**

Comment (eddiejaoude) [reply data/Laxmankohar.json:28]: Don't worry, the Prettier GitHub Action will format the file

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Comment by Laxmankohar** [comment]
> Looks good 👍

Thanks @eddiejaoude

**Pull request merged**